starter-code/contact-app/src/app/contact-list/contact-list.component.ts

```diff
@@ -8,17 +8,32 @@ import contactList from '../contacts';
 })
 export class ContactListComponent implements OnInit {
   contacts: Object[];
-  newContact: Object = {};
+  newContact: Object = {
+    name: null,
```

Comment:
You are working with angular 2.4, in the latest version 5.0, you would get a error, as you did not defined the keys in the ts file, still you are refering to for example newContact.name in the html file. To fix this, see my code below. To reset the form again after you need to reset each key.
```
import { Component, OnInit } from "@angular/core";
import contactList from "../contacts";

@Component({
  selector: "app-contact-list",
  templateUrl: "./contact-list.component.html",
  styleUrls: ["./contact-list.component.css"]
})
export class ContactListComponent implements OnInit {
  contacts: object[];
  newContact: {
    name: string,
    email: string,
    phoneNumber: string,
    image: string
  };
  constructor() { }

  ngOnInit() {
    this.contacts = contactList;
  }
  addContact(contact) {
    console.log("Add contact has been called");
    this.contacts.push(contact);
    this.newContact = {
      name: "",
      email: "",
      phoneNumber: "",
      image: ""
    };
  }
}
```